corehq/apps/linked_domain/management/commands/add_app_id_to_linked_reports.py

```diff
@@ -0,0 +1,50 @@
+from django.core.management import BaseCommand, CommandError
+
+from corehq.apps.linked_domain.applications import get_downstream_app_id
+from corehq.apps.linked_domain.models import DomainLink
+from corehq.apps.userreports.dbaccessors import get_report_configs_for_domain
+from corehq.apps.userreports.models import DataSourceConfiguration, ReportConfiguration
+
+
+def migrate_linked_reports(upstream_domain=None):
+    if upstream_domain:
+        domain_links = DomainLink.objects.filter(master_domain=upstream_domain)
+    else:
+        domain_links = DomainLink.objects.filter(deleted=False)
```

Comment:
Can DomainLinks be undeleted? Should deleted ones be populated just to be safe?

Reply:
Yes thank you I meant to highlight this as a question I had. I think migrating deleted ones makes sense. I can't think of any downsides to doing this.